doc/techref/fonts.md

```diff
@@ -0,0 +1,36 @@
+# Supported Fonts
+
+PyGMT supports the standard 35 PostScript fonts. The table below lists the 35 fonts with
+font number and the font name.
+
+| Font Number | Font Name                  | Font Number | Font Name                  |
```

Comment:
The table is added so that users can directly copy the font name. The figure is added so that users can choose the suitable font visually.